client.go

```diff
@@ -9,13 +9,13 @@ import (
 // Client represents an instance of the 1Password Go SDK client.
 type Client struct {
 	config  internal.ClientConfig
-	Secrets SecretsAPI
-	Items   ItemsAPI
-	Vaults  VaultsAPI
+	Secrets SecretsSource
+	Items   ItemsSource
+	Vaults  VaultsSource
 }
 
 func initAPIs(client *Client, inner internal.InnerClient) {
-	client.Secrets = NewSecretsSource(inner)
-	client.Items = NewItemsSource(inner)
-	client.Vaults = NewVaultsSource(inner)
+	client.Secrets = *NewSecretsSource(inner)
+	client.Items = *NewItemsSource(inner)
+	client.Vaults = *NewVaultsSource(inner)
```

Comment:
What happened here?

Reply:
In order to enable nested functions in the SDKs, we removed the interfaces that were priorly used as dependencies within the `Client` and are adding structs directly, instead.

This is because, in order to be able to do `client.items.shares.(...)`, `items` can't be an interface, since interfaces can't do fields.

Reply:
Oh I see. The interfaces were meant for testing purposes in order to mock some e.g. item retrieval dependencies, when wanting to test some logic specific to the Go SDK code. However, the SDKs evolved to not have that much user-facing logic after all, so I'm fine with removing the interfaces.